[Doc] Add from_unixtime_ms english doc

chaoyli opened the pull request:
## Why I'm doing:
Add docs for the from_unixtime_ms

## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.5
  - [x] 3.4
  - [ ] 3.3
  - [ ] 3.2
  - [ ] 3.1

alvin-celerdata commented on docs/en/sql-reference/sql-functions/date-time-functions/from_unixtime_ms.md:
```diff
@@ -0,0 +1,58 @@
+| 2007-12-01 00:30:19          |
++------------------------------+
+
+mysql> SELECT from_unixtime_ms(1196440219123);
++------------------------------+
+| from_unixtime_ms(1196440219123) |
++------------------------------+
+| 2007-12-01 00:30:19          |
++------------------------------+
```
I don't think the result is what was expected.

chaoyli committed:
[BugFix] Redact credentials in audit log when SQL parsing fails

When SQL parsing fails due to syntax errors, credentials in the original SQL statement
are exposed in the audit log. This fix adds credential redaction for unparsed SQL
statements to prevent security information leakage.

Changes:
- Add SqlCredentialRedactor utility to redact sensitive credentials from SQL strings
- Update ConnectProcessor.auditAfterExec to use redaction when parsedStmt is null
- Add comprehensive unit tests for credential redaction

The fix ensures that credentials like aws.s3.access_key, aws.s3.secret_key, and other
sensitive keys are replaced with *** in audit logs when SQL parsing fails.
